src/modules/elements.js

```diff
@@ -0,0 +1,19 @@
+import dltIcon from '../assets/icons/bin.png';
+
+const element = (obj) => {
+  const li = document.createElement('li');
+  li.classList = `${obj.index}`;
+  const checkbox = document.createElement('input');
+  checkbox.type = 'checkbox';
+  checkbox.classList = 'checkbox';
+  const textArea = document.createElement('textarea');
+  textArea.classList = 'childTextArea';
+  textArea.value = `${obj.inputValue}`;
+  const img = document.createElement('img');
+  img.classList = 'dltIcon';
+  img.src = dltIcon;
+  li.append(checkbox, textArea, img);
+  document.querySelector('ul').appendChild(li);
+};
+
```

Comment:
- [ ] Kindly ensure you implement the functionality for editing a task properly. The change only persists on the page, but not in the local storage so when I reload the page the old description still appears. Kindly fix this.